src/libraries/System.Formats.Cbor/ref/System.Formats.Cbor.csproj

```diff
@@ -1,13 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
+    <TargetFrameworks>net5.0</TargetFrameworks>
```

Comment:
why?

Reply:
Just saw we shipped the package with net5 yesterday. Was asking as I thought that you handled these configuration changes when you updated the tfm to net6.

Reply:
Yeah. it seems like it slipped through as the package index wasn't complete and hence it didn't fail because this package was knew and there were not inferred supported TFMs from previous packages, and now the build was failing saying that there was no compile assets for 5.0 for this package which was supported by the previous package.

Reply:
That makes sense, thanks for explaining.